app/api/resources/templates/interstitial.j2

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
```

Comment:
- Where is the `html` tag? Also, let's have a language on it.
- Let's at least have a page title.